content/en/news/releases/1.15.x/announcing-1.15.2/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.15.2
+linktitle: 1.15.2
+subtitle: Patch Release
+description: Istio 1.15.2 patch release.
+publishdate: 2022-10-07
+release: 1.15.2
+---
+
+This release includes security fixes in Go 1.19.2 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+This release contains bug fixes to improve robustness.
+This release note describes what is different between Istio 1.15.2 and Istio 1.15.2.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0, disabling the timeout. Restored previous behavior to using Envoy's default value of 1 hour. ([Issue #41114](https://github.com/istio/istio/issues/41114))
+
+- **Fixed** the `gateway-api` integration to read `v1beta1` resources for `HTTPRoute`, `Gateway`, and `GatewayClass`. Users of the `gateway-api` must be on version 0.5.0 or higher before upgrading Istio.
```

Comment:
Can we just remove the last sentence `Users of the `gateway-api` must be on version 0.5.0 or higher before upgrading Istio.`? https://istio.slack.com/archives/C036AKKFZ7Z/p1665081516268089 is a pointer to were I brought this up.

Reply:
The 1.13.9 RN (https://github.com/istio/istio.io/pull/12053) as well as the 1.14.5 RN changed this to `Fixed the gateway API integration to not fail when the v1alpha2 version is removed.`.